### ISSUE TITLE
[Screen Reader - Make Code - Extensions] Narrator focus does not synchronized with keyboard focus on all controls under 'More' menu item.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2362]
**User Experience:**
Screen reader user might get difficulty in navigation to interact with control if narrator focus doesn’t land on any control along with keyboard focus as user won't be able to reach out that control. Not only this, they won't be able to get relevant information in respect of that control and also he/she will skip further.

**Repro-Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "More" option appearing in header section and select it.
6. Navigate to "Extensions, Print, Delete project, Report abuse etc." option.
7. Verify narrator focus along with keyboard focus on all controls(Extensions, Print, Delete project, Report abuse) or not.

**Actual Result:**
Narrator focus is not synchronized with keyboard focus on all controls like (Extensions, Print, Delete project, Report abuse) present under 'More' menu item on header section, due to this screen reader user is unable to interact with the control and also won't be able to take action on such control. Not only this, he/she would miss the functionality of such control.

**Expected Result:**
Narrator focus should synchronized with keyboard focus on all controls like (Extensions, Print, Delete project, Report abuse) present under 'More' menu item on header section, so that screen reader user could easily interact with the control for further navigation.

**Note:**
1. This issue doesn't exist with Chrome + JAWS.
2. This issue doesn't exist with Chrome + NVDA(Browse mode).

**MAS Reference:**
[MAS 4.2.10 - Tracking of Focus and Selection Attributes](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad})

**Labels**
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS4.2.10, #Win10-Edge, #accessibility

[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

[MAS4.2.10_More_Narrator focus does not sync with keyboard focus.zip](https://github.com/microsoft/pxt-microbit/files/3476388/MAS4.2.10_More_Narrator.focus.does.not.sync.with.keyboard.focus.zip)

**Comment by abchatra** [comment]
This should be fixed in latest edge as it uses chromium.
